web/src/games/bingo/utils.ts

```diff
@@ -17,3 +17,22 @@ export function inferActivePlayers(G: IGameState, playerID: string) {
   const shoutCallPlayers = Object.keys(G.players).filter((p) => G.players[p].shoutCount > 0 && p !== playerID);
   return [playerID, ...shoutCallPlayers].slice(0, MAX_BACKOFF_CANDIDATES);
 }
+
+export function gameLocalStore(action: string, matchCode: string, key: string, value: any = null) {
+  if (typeof window !== 'undefined') {
```

Comment:
You can replace this with a guard statement:
```
if (typeof window === 'undefined') {
  return;
}
```

This way, you have less nesting which makes the code easier to read.

Reply:
done in new commit